CVE-2017-9805/cvex.yml

```diff
@@ -0,0 +1,11 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+  command:
+    - "echo 'hello world' | sudo tee /opt/hello > /dev/null"
```

Comment:
Why do we need to execute this command?

Reply:
For debug purposes; it is not required to make the exploit.
Perhaps, it should be deleted?

Reply:
Having something for debugging purposes is fine, but does it actually help to see more? If it does, leave it.